packages/data-stores/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/data-stores",
-	"version": "1.0.0-alpha.1",
```

Comment:
I'd suggest dropping the `-alpha.0` suffix. It is not very common in the ecosystem and the precedence rules are not trivial (https://semver.org/#spec-item-11). 

I suggest we go with just `2.0.0`, and if there are bugs we release it as `2.0.1` and so on.